runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -1891,6 +1902,32 @@ class INTERPRETER_CLASS
 				}
 				break;
 			}
+#if defined(J9VM_OPT_MICROJIT)
+			if (mjitExtra) {
```

Comment:
Why introduce a new copy of the `preCount` variable?  Can you reuse the existing one?

```suggestion
			if (mjitExtra) {
```